src/lib/spinel/spinel.h

```diff
@@ -4349,6 +4353,18 @@ enum
      */
     SPINEL_PROP_RCP_API_VERSION = SPINEL_PROP_RCP__BEGIN + 0,
 
+    /// Min host RCP API Version number
+    /** Format: `i` (read-only)
+     *
+     * Required capability: SPINEL_CAP_RADIO and SPINEL_CAP_RCP_MIN_HOST_API_VERSION.
+     *
+     * This property gives the minimum host RCP API Version number.
+     *
+     * Please see "Spinel definition compatibility guideline" section.
+     *
+     */
+    SPINEL_PROP_RCP_MIN_HOST_API_VERSION = SPINEL_PROP_RCP__BEGIN + 1,
```

Comment:
Is it sufficient to check presence of SPINEL_CAP_RCP_MIN_HOST_API_VERSION to determine whether this is supported or not? Otherwise, it seems every time we add a new property, we need a new capability. Thoughts?

Reply:
Yes. We can check presense of the related `CAP`. 
We provide two ways in the guideline: Either add a cap or host code needs to handle NOT_FOUND error accordingly.

In the case here, I prefer to use `CAP` to be similar to existing `CAP_API_VERSION`.

Reply:
For consistency, should we switch both RCP API and RCP_MIN_HOST_API to this NOT_FOUND approach? It seems we can deprecate the `SPINEL_CAP_RCP_API_VERSION` capability without harming the compatibility.

Reply:
I prefer not to change existing code and behavior while adding new property and follow the existing (and tested) model. This would be more consistent :)

If you feel passionate about it, feel free to submit a PR later to change this. But I advice caution. I am not sure if the NOT_FOUND model is used and tested enough. So there may be risks  (things we may not see about it). I would highly recommend adding test-case to validate its behavior if you change it.